src/main/java/com/microsoft/graph/requests/extensions/DriveItemDeltaCollectionRequestBuilder.java

```diff
@@ -33,7 +33,10 @@
      * @param token the token
      */
     public DriveItemDeltaCollectionRequestBuilder(final String requestUrl, final IBaseClient client, final java.util.List<? extends Option> requestOptions, final String token) {
-        super(requestUrl, client, requestOptions, token);
+        super(requestUrl, client, requestOptions);
+        if(token != null) {
```

Comment:
Shouldn't this logic be moved into the original constructor being called? I know it's not in the extensions package but it should be fixed there instead.

Reply:
It would be cleaner to do it that way. BaseDriveItemDeltaCollectionRequestBuilder has a comment stating that it's generated code, and I'm not sure how or when that code is generated.